src/hello-world/hello.txt

```diff
@@ -0,0 +1,6 @@
+console.log('yo!')
+console.log('No, Yao!');
+console.log('Hello again!');
+console.log('Hello, goodbye!');
+
+// hello
```

Comment:
I'm guessing we don't want this to be in the PR 😅

Reply:
Deleted.